code/modules/mob/living/simple_animal/vox.dm

```diff
@@ -33,7 +33,10 @@
 	var/quills = 3
 
 /mob/living/simple_animal/vox/armalis/death(var/gibbed = FALSE)
-	..(TRUE)
+	if (gibbed)
+		..()
+		return
```

Comment:
```suggestion
		return ..()
```
?

Reply:
`death()` never returns anything so is basically void return type wise. Making these separate makes that clear.

Reply:
Ok